Jakefile.js

```diff
@@ -954,7 +954,7 @@ var nodeServerInFile = "tests/webTestServer.ts";
 compileFile(nodeServerOutFile, [nodeServerInFile], [builtLocalDirectory, tscFile], [], /*useBuiltCompiler:*/ true, { noOutFile: true, lib: "es6" });
 
 desc("Runs browserify on run.js to produce a file suitable for running tests in the browser");
-task("browserify", ["tests", builtLocalDirectory, nodeServerOutFile], function() {
+task("browserify", ["tests", run, builtLocalDirectory, nodeServerOutFile], function() {

```

Comment:
I think on [this line](https://github.com/Microsoft/TypeScript/blob/master/Gulpfile.ts#L748) change `[servicesFile]` to `[run, servicesFile]` (or maybe `[servicesFile, run]`).